### ISSUE TITLE
Gui crashes when doing a reverse swap and immediately clicking the history tab

### ISSUE THREAD

**Issue by bitromortac** [spesmilo/electrum#7137]
To reproduce (I did it on testnet):
Do a reverse swap (ln->onchain).
After LN payment succeeds, immediately click the history tab.

```Traceback (most recent call last):
  File "/home/user/electrum/electrum/gui/qt/util.py", line 762, in showEvent
    self.update()
  File "/home/user/electrum/electrum/gui/qt/history_list.py", line 473, in update
    self.hm.refresh('HistoryList.update()')
  File "/home/user/electrum/electrum/util.py", line 412, in <lambda>
    return lambda *args, **kw_args: do_profile(args, kw_args)
  File "/home/user/electrum/electrum/util.py", line 408, in do_profile
    o = func(*args, **kw_args)
  File "/home/user/electrum/electrum/gui/qt/history_list.py", line 268, in refresh
    transactions = wallet.get_full_history(
  File "/home/user/electrum/electrum/util.py", line 412, in <lambda>
    return lambda *args, **kw_args: do_profile(args, kw_args)
  File "/home/user/electrum/electrum/util.py", line 408, in do_profile
    o = func(*args, **kw_args)
  File "/home/user/electrum/electrum/wallet.py", line 947, in get_full_history
    fiat_fields = self.get_tx_item_fiat(tx_hash=txid, amount_sat=value, fx=fx, tx_fee=item['fee_sat'])
KeyError: 'fee_sat'
Aborted (core dumped)```

**Comment by SomberNight** [comment]
Looking at the last line, did it segfault?
Or is it simply the crash reporter that appeared?

**Comment by ecdsa** [comment]
fixed in 73482bb
